experiments/generic/disk_loss/disk_loss_ansible_logic.yml

```diff
@@ -0,0 +1,107 @@
+- hosts: localhost
+  connection: local
+
+  vars:
+    cloud_platform: "{{ lookup('env','CLOUD_PLATFORM') }}"
+    c_experiment: disk-loss
+    a_check: "{{ lookup('env', 'APP_CHECK') }}"
+    a_ns: "{{ lookup('env','APP_NAMESPACE') }}"
+    c_ns: "{{ lookup('env','CHAOS_NAMESPACE') }}"
+    a_label: "{{ lookup('env','APP_LABEL') }}"
+
+  tasks:
+
+    - block:
+        - include: disk_loss_ansible_prerequisites.yml
+
+        - include_vars:
+            file: chaosutil.yml 
+
+        ## GENERATE EXP RESULT NAME
+        - block:
+            - name: Construct chaos result name (experiment_name)
+              set_fact:
+                c_experiment: "{{ lookup('env','CHAOSENGINE') }}-{{ c_experiment }}"
+          when: lookup('env','CHAOSENGINE')
+
+        # RECORD START-OF-EXPERIMENT IN LITMUSCHAOS RESULT CR
+        - include_tasks: "{{ update_chaos_result }}"
+          vars:
+            status: 'SOT'
+            namespace: "{{ c_ns }}"
+
+        ## PRE-CHAOS APPLICATION LIVENESS CHECK
+        - name: Verify that the AUT (Application Under Test) is running
+          include_tasks: "{{ pod_status }}"
+          vars:
+            app_ns: "{{ a_ns }}"
+            app_lkey: "{{ a_label.split('=')[0] }}"
```

Comment:
Can you directly pass label without splitting it into key and value?

Reply:
It will be a conflict if I do this because your PR is in the review. So, I left as it is.